starter-code/src/viking.js

```diff
@@ -1,11 +1,97 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damage) {
+    this.health -= damage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super();
+    this.name = name;
+    this.health = health;
+    this.strength = strength;
```

Comment:
Tu peux faire passer dans super health et strength => super(health, strength) ce qui te permet d'etre sur que tu récupères bien les propriétés du Solider. Du coup pas besoin d'écrire les lignes: this.health = health; this.strength = strength;